src/partitionBy.js

```diff
@@ -0,0 +1,2 @@
+var _partitionBy = require('./internal/_partitionBy');
+module.exports = _partitionBy;
```

Comment:
I think the point is that there are now two equal references to the same function: '`_partitionBy`' and '`partitionBy`'.  We do this sort of wrapping of internal functions to add currying or `checkForMethod`.  But here there seems to be no point.  It looks as though the internal version could simply be moved into the public implementation.